code/modules/uplink/uplink_item.dm

```diff
@@ -134,6 +134,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/item/weapon/storage/backpack/dufflebag/syndie/ammo/fireteam
 	cost = 24 // normally ??
 
+/datum/uplink_item/nukeoffer/firestarter
+	name = "Spetsnaz Pyro bundle"
+	desc = "For systematic suppression of carbon lifeforms in close range: Contains a specialist Pyrotechnic equipment, foreign pistol, two magazines, a pipebomb, and a stimulant syringe."
+	item = /obj/item/weapon/storage/backpack/dufflebag/syndie/firestarter
+	cost = 20
```

Comment:
way too cheap even for the killpack itself, let alone the free suit + minibomb
double the price at the very least